lib/recurly/pricing/attach.js

```diff
@@ -60,7 +60,7 @@ export function attach (el) {
     }
 
     if (elems.coupon && (target('coupon') || target('plan'))) {
-      pricing = pricing.coupon(dom.value(elems.coupon)).then(null, ignoreBadCoupons);
+      pricing = pricing.coupon(dom.value(elems.coupon).trim()).then(null, ignoreBadCoupons);
```

Comment:
This was a little thing requested to trim coupon codes, since they cannot contain whitespace anyway, and email copypasta sometimes generates lots of whitespace.